refinery/core/serializers.py

```diff
@@ -47,6 +49,16 @@ def get_owner(self, analysis):
     def get_data_set_uuid(self, analysis):
         return analysis.data_set.uuid
 
+    def get_creation_date(self, analysis):
```

Comment:
The provenance client side should be updated to use time_start instead of adding another field to the serializer with the same functionality.